ui/packages/catalog-ui-search/package.json

```diff
@@ -28,7 +28,8 @@
     "@connexta/ace": "git://github.com/connexta/ace.git#v0.0.7",
     "chai": "4.1.1",
     "enzyme": "3.3.0",
-    "enzyme-adapter-react-16": "1.1.1",
+    "enzyme-adapter-react-16": "1.5.0",
+    "js-beautify": "1.7.5",
```

Comment:
please remove the dependency

Reply:
Versions prior to `1.5.0` have a documented bug with I was encountering:
https://github.com/airbnb/enzyme/issues/1620